packages/xml-views-validation/test/validators/element/type-of-aggregation-spec.ts

```diff
@@ -0,0 +1,97 @@
+import { partial } from "lodash";
+import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+import { generateModel } from "@ui5-language-assistant/test-utils";
+import {
+  assertNoIssues as assertNoIssuesBase,
+  assertSingleIssue as assertSingleIssueBase,
+} from "../../test-utils";
+import { validateAggregationType } from "../../../src/validators/elements/type-of-aggregation";
+import {
+  getMessage,
+  INVALID_AGGREGATION_TYPE,
+} from "../../../src/utils/messages";
+
+describe("the type aggregation validation", () => {
+  let ui5SemanticModel: UI5SemanticModel;
+
+  before(async () => {
+    ui5SemanticModel = await generateModel({ version: "1.74.0" });
+  });
+
+  context("true positive scenarios", () => {
+    let assertSingleIssue: (xmlSnippet: string, message: string) => void;
+    before(() => {
+      assertSingleIssue = partial(
+        assertSingleIssueBase,
+        ui5SemanticModel,
+        {
+          element: [validateAggregationType],
+        },
+        "InvalidAggregationType",
+        "error"
+      );
+    });
+
+    it("will detect mismatch of class to explicit aggregation type", () => {
+      assertSingleIssue(
+        `<mvc:View xmlns:uxap="sap.uxap" xmlns:m="sap.m"
+            xmlns:mvc="sap.ui.core.mvc"
+            xmlns="sap.ui.commons">
+            <m:Panel>
+              <m:headerToolbar>
+                <🢂Toolbar🢀></Toolbar>
+              </m:headerToolbar>
+            </m:Panel>
+          </mvc:View>`,
+        getMessage(INVALID_AGGREGATION_TYPE, "Toolbar", "Toolbar")
+      );
+    });
+
+    it("will detect mismatch of class to explicit aggregation of 'UI5Interface' type", () => {
```

Comment:
Should this description include the term "default aggregation"?
Also I do not think this is "explicit" case

Reply:
again this is not resolved, this is not an explicit aggregation case in this snippet
but instead an implicit (default) aggregation of Panel